.github/workflows/issue-notifications.yaml

```diff
@@ -51,3 +51,12 @@ jobs:
             const action = require('${{ github.workspace }}/.github/scripts/notifyAboutNewComponent.js')
             await action({github, context, core})
 
+      - name: "Icon request notification"
+        if: github.event.label.name == 'calcite-ui-icons'
+        uses: actions/github-script@v7
+        env:
+          ICONS_TEAM: ${{secrets.ICON_LEADS}}
+        with:
+          script: |
+            const action = require('${{ github.workspace }}/.github/scripts/notifyAboutIconRequest.js')
+            await action({github, context, core})
```

Comment:
can you use the `yml` extension instead of `yaml` to align with the rest of the workflows?